WcaOnRails/spec/features/create_committee_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.feature "Create Committees", js: true do
+  let(:admin) { FactoryGirl.create(:admin) }
+
+  before :each do
+    sign_in admin
+  end
+
+  it 'can create committee when signed in as admin' do
+    visit "/committees/new"
+
+    fill_in "Name", with: "Random Ideas Committee"
+    fill_in "Email", with: "random@worldcubeassociation.org"
+    fill_in "Duties", with: "Come up with random ideas about cubing."
+
+    click_button "Create a New Committee"
+
+    new_committee = Committee.find_by_slug("random-ideas-committee")
+    expect(new_committee.name).to eq "Random Ideas Committee"
+    expect(new_committee.duties).to eq "Come up with random ideas about cubing."
```

Comment:
As it's feature spec better approach would be to go to committees index and make expectations on the page content rather than checking database.